crates/proc-macro-srv/src/lib.rs

```diff
@@ -66,13 +68,18 @@ impl ProcMacroSrv {
         // FIXME: replace this with std's scoped threads once they stabilize
         // (then remove dependency on crossbeam)
         let result = crossbeam::scope(|s| {
-            let res = s
+            let res = match s
+                .builder()
+                .stack_size(EXPANDER_STACK_SIZE)
+                .name(format!("macro_expander_{}", task.macro_name))
```

Comment:
```suggestion
                .name(task.macro_name.clone()))
```

See #9532.

It's not super important, but can you also squash these? It's a bit weird having 3-4 commits for this :grinning:.